SrProj/Static/partials/patronCheckIn.html

```diff
@@ -0,0 +1,142 @@
+﻿
+<span class="topMargin">
+    <div class="ui centered grid">
+        <div class="nine wide column">
+            <div class="ui form segment">
+                <div class="two fields">
+                    <div class="field">
+                        <label>First name</label>
+                        <input type ="text" placeholder="First Name"/>
+                    </div>
+                    <div class="field">
+                        <label>Middle name</label>
+                        <input type="text" placeholder="Middle Name" />
+                    </div>
+                    <div class="field">
+                        <label>Last name</label>
+                        <input type="text" placeholder="Last Name">
+                    </div>
+            </div>
+                <!--date of birth-->
+                <div class="date field">
+                    <label>Birthday</label>
+                    <input type="text" placeholder="xx/xx/xxxx">
+                </div>
+                <!--Gender & Ethnicity fields -->
+                <div class="two fields">
+                <div class="field">
+                    <label>Gender</label>
+                        <select class="ui dropdown">
+                            <option value="">Gender</option>
```

Comment:
Little weird to do this with 0 as the bottom value. Typically your first option is value (index) 0. But this will be overwritten when wired up since it'll be coming from the app.